dnf-docker-test/features/repoquery-3.feature

```diff
@@ -0,0 +1,124 @@
+Feature: Test for dnf repoquery, options --all, --installed, --available, --upgrades, --recent, --extras, --repo 
+ repo base: TestA-1 TestB-1
+ repo ext: XTest-2 TestA-2 TestB-2
+
+  @setup
+  Scenario: Setup (create test repos)
+      Given repository "base" with packages
+         | Package      | Tag      | Value     |
+         | TestA        | Version  | 1         |
+         | TestB        | Version  | 1         |
+        And repository "ext" with packages
+         | Package      | Tag      | Value     |
+         | XTest        | Version  | 2         |
+         | TestA        | Version  | 2         |
+         | TestB        | Version  | 2         |
+
+  # --extras: installed pkgs, not from known repos
+  Scenario: dnf repoquery --extras (when there are such pkgs)
+       When I successfully run "rpm -i TestA-1*.rpm" in repository "base"
+        And I successfully run "dnf repoquery --extras"
+       Then the command stdout should match regexp "TestA.*1-1"
```

Comment:
I would recommend also negative step like ``Then the command stdout should not match regexp "TestB.*1-1"``